content/getting-started/php.textile

```diff
@@ -0,0 +1,215 @@
+mkdir ably-php-quickstart
+cd ably-php-quickstart
+
+# Initialize Composer (if you don't have a composer.json yet)
+composer init
+
+# Install the Ably PHP SDK
+composer require ably/ably-php
+```
+
+<aside data-type='note'>
+<p>The code examples in this guide include a demo API key. If you wish to interact with the Ably CLI and view outputs within your Ably account, ensure that you replace them with your own API key.</p>
+</aside>
+
+
+h2(#step-1). Step 1: Configure the Ably Rest SDK client
+
+Clients can make REST API requests to Ably when they instantiate a REST SDK instance. This enables them to publish messages to channels and retrieve channel information.
+
+* Create a @get_started.php@ file in your project and add the following code to instantiate the SDK and make requests to Ably. At the minimum you need to provide an authentication mechanism. Using an API key directly is fine for server-side environments like PHP where the key remains secure. However, for frontend clients, you should use token authentication instead of exposing API keys. In this example, the @clientId@ will be the name of the client sending the message:
+
+```[php]
+<?php
+
+require 'vendor/autoload.php';
+
+use Ably\AblyRest;
+
+$ably = new AblyRest(['key' => '{{API_KEY}}', 'clientId' => 'my-first-client']);
+```
+
+h2(#step-2). Step 2: Generate a JWT
+
+The Ably PHP SDK provides token authentication for secure client connections. Authentication with a JWT is recommended over using API keys directly in production because they can be scoped to specific capabilities and have expiration times. This is essential when authenticating frontend clients where embedding API keys would expose them publicly. With PHP the API key would be considered securely stored as it is on the server side.
+
+This section will show you how to build a JWT with the Ably API key credentials embedded, which you can then return to your frontend clients, for example from an API endpoint.
+
+First install the Firebase JWT library, which is used to create and verify JSON Web Tokens (JWTs):
+
+```[bash]
+composer require firebase/php-jwt
+```
+
+Import the necessary classes at the top of your @get_started.php@ file:
+
+```[php]
+use Firebase\JWT\JWT;
+use Firebase\JWT\Key;
+```
+
+Then create a new JWT with the following code:
+
+```[php]
+$currentTime = time();
+$payload = [
+    'iat' => $currentTime,
+    'exp' => $currentTime + 3600,
+    'x-ably-capability' => '{"*":["*"]}'
+];
+
+$apiKeyCredentials = explode(':', '{{API_KEY}}');
+$jwt = JWT::encode(
+    $payload,
+    $apiKeyCredentials[1],
+    'HS256',
+    $apiKeyCredentials[0]
+);
+
+echo $jwt;
+```
+
+To use this JWT for authenticating frontend clients, you would typically expose it through an API endpoint that your frontend application can call. For example, you might create an endpoint like @/auth/token@ that returns this JWT. Your frontend clients can then use this JWT when instantiating their Ably client:
+
+```[javascript]
```

Comment:
Unfortunately this will now add JS to the dropdown of languages to view the page in. 

I think the best way to get around this is to set the language to something like shell, JSON or PHP and then use the `fixed` property to hide the language; `<Code fixed="true">`